docs/events/workshops/posts/2024-workshop.md

```diff
@@ -1,7 +1,18 @@
 ---
-date: 2024-02-13
```

Comment:
This will change the URL you link to in the announcement bar, probably leave this one out?